app/src/main/java/com/simply/birthdayapp/auth/registration/presentation/RegistrationViewModel.kt

```diff
@@ -140,4 +148,29 @@ class RegistrationViewModel : ViewModel() {
         _isRegisterEnabled.value =
             (nameError.value == null) && (surnameError.value == null) && (emailError.value == null) && (passwordError.value == null) && (repeatedPasswordError.value == null) && _name.value.isNotEmpty() && _surname.value.isNotEmpty() && _email.value.isNotEmpty() && _password.value.isNotEmpty() && _repeatedPassword.value.isNotEmpty()
     }
+
+
+    fun signUp(signUpInput: SignUpInput) {
+        viewModelScope.launch {
+            _uiState.value = SignUpUiState.Loading
+
+            val result = signUpUseCase.invoke(signUpInput)
+            _uiState.value = when (result) {
+                is Result.Success -> SignUpUiState.Success("Account created successfully!")
```

Comment:
please do not hardcode strings in viewmodel or below layers, consider to use enums or states and handle it in Screen